independent-publisher-connectors/WebexIntegration/apiDefinition.swagger.json

```diff
@@ -0,0 +1,814 @@
+                  "title": "",
+                  "default": "Europe/London"
+                },
+                "enabledAutoRecordMeeting": {
+                  "type": "boolean",
+                  "description": "enabledAutoRecordMeeting",
+                  "title": "",
+                  "enum": [
+                    true,
+                    false
+                  ],
+                  "default": false
+                },
+                "allowAnyUserToBeCoHost": {
+                  "type": "boolean",
+                  "description": "allowAnyUserToBeCoHost",
+                  "title": "",
+                  "enum": [
+                    true,
+                    false
+                  ],
+                  "default": false
+                }
+              },
+              "default": {
+                "title": "Sample Title",
+                "agenda": "Sample Agenda",
+                "password": "A@ssword123",
+                "start": "{{_start_time}}",
+                "end": "{{_end_time}}",
+                "timezone": "{{TIMEZONE_STRING}}",
+                "enabledAutoRecordMeeting": false,
+                "allowAnyUserToBeCoHost": false
+              },
+              "required": [
+                "title",
+                "start",
+                "end"
+              ]
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "201": {
+            "description": "Created"
+          },
+          "202": {
+            "description": "Accepted"
+          },
+          "204": {
+            "description": "No Content"
+          },
+          "400": {
+            "description": "Bad Request"
+          },
+          "404": {
+            "description": "Not Found"
+          },
+          "401": {
+            "description": "Unauthorized"
+          },
+          "403": {
+            "description": "Forbidden"
+          },
+          "405": {
+            "description": "Method Not Allowed"
+          },
+          "409": {
+            "description": "Conflict"
+          },
+          "410": {
+            "description": "Gone"
+          },
+          "415": {
+            "description": "Unsupported Media Type"
+          },
+          "423": {
+            "description": "Locked"
+          },
+          "429": {
+            "description": "Too Many Requests"
+          },
+          "500": {
+            "description": "Internal Server Error. Unknown error occurred"
+          },
+          "502": {
+            "description": "Bad Gateway"
+          },
+          "503": {
+            "description": "Service Unavailable"
+          },
+          "default": {
+            "description": "Operation Failed."
+          }
+        }
+      }
+    },
+    "/meetingInvitees": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "OK"
```

Comment:
@ocarhui apologies for not catching this from the beginning. here you are missing closing curly braces "}". the closing tab for "200" responses. 
this error is occurring multiple times in the swagger file, please double check the syntax for all the responses in the swagger file. 
Thank you so much again.